### ISSUE TITLE
Improve accessibility (and SEO)

### ISSUE THREAD

**Issue by radex02** [nk-crew/awb#45]
It would improve the plugin's accessibility if the Iframe (when using Youtube or Vimeo) had a "title" attribute and if the poster images had an "alt" attribute.

**Comment by radex02** [comment]
Also, there should be a aria-hidden="true" on the iframe
